internal/proxy/client_test.go

```diff
@@ -148,7 +148,7 @@ func TestInfo(t *testing.T) {
 
 	sockPath := fmt.Sprintf("unix://%s/echo_%v.sock", t.TempDir(), tmrand.Str(6))
 	logger := log.NewNopLogger()
-	client, err := abciclient.NewClient(logger, sockPath, SOCKET, true)
+	client, err := abciclient.NewClient(logger, sockPath, SOCKET)
```

Comment:
I see what you mean. Though OnStart returns in the successful path (and retries forever in the "there's no server path") but that's not the case in this (or any test).

Reply:
Yeah, it's the "retries forever" path that I mean. It's not a huge deal, but it means that the test will time out rather than reporting an error in case that happens.